README.md

```diff
@@ -29,7 +29,8 @@ Let's take a look at all the neat things we provide:
 ✅ Provides Helpful Methods <br>
 ✅ Typed and DRY `ControlValueAccessor` <br>
 ✅ Typed `FormBuilder` <br>
-✅ Persist the form's state to local storage
+✅ Persist the form's state to local storage <br>
+✅ Compatible with Angular versions 8+
```

Comment:
It's not the right place

Reply:
happy to update, where do you think this information would best fit?

Reply:
perhaps a 'compatibility' section at the end right after the 'migration' section? that was the other one of my two first thoughts where to put it

Reply:
Sounds good

Reply:
Also, add yourself as a contributor.

Reply:
updated, cheers!